frontend/public/components/pod.jsx

```diff
@@ -289,15 +285,17 @@ PodList.displayName = 'PodList';
 
 const filters = [{
   type: 'pod-status',
-  selected: ['Running', 'Pending', 'Terminating', 'CrashLoopBackOff'],
-  reducer: podPhase,
+  selected: [ 'Running', 'Pending', 'Terminating', 'CrashLoopBackOff', 'Unknown' ],
+  reducer: podPhaseFilterReducer,
   items: [
-    {id: 'Running', title: 'Running'},
-    {id: 'Pending', title: 'Pending'},
-    {id: 'Terminating', title: 'Terminating'},
-    {id: 'CrashLoopBackOff', title: 'CrashLoopBackOff'},
-    {id: 'Completed', title: 'Completed'},
-  ],
+    { id: 'Running', title: 'Running' },
+    { id: 'Pending', title: 'Pending' },
+    { id: 'CrashLoopBackOff', title: 'CrashLoopBackOff' },
+    { id: 'Succeeded', title: 'Succeeded' },
+    { id: 'Failed', title: 'Failed' },
+    { id: 'Terminating', title: 'Terminating' },
```

Comment:
Tempted to suggest moving terminating back to where it was, too (before crash looping pods). Then all the statuses that are deselected by default are together on the right.